client/components/jetpack-sites-dashboard/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { DataViewsState } from './types';
+
+export const initialSitesViewState: DataViewsState = {
```

Comment:
This is the initial state required by [@wordpress/dataviews](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dataviews/), plus a `selectedSiteId` that we use to open and close the site preview pane.